dp_check/dp_check.go

```diff
@@ -38,17 +38,20 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/vishvananda/netlink"
 	"google.golang.org/grpc"
 	lbpb "google.golang.org/grpc/balancer/grpclb/grpc_lb_v1"
 	"google.golang.org/grpc/connectivity"
 	"google.golang.org/grpc/credentials/alts"
 )
 
 var (
-	service      = flag.String("service", "", "The public DirectPath-enabled DNS of the service to check")
-	infoLog      = log.New(os.Stderr, "INFO: ", log.Ldate|log.Ltime|log.Lshortfile)
-	failureCount int
-	runningOS    = runtime.GOOS
+	service                = flag.String("service", "", "The public DirectPath-enabled DNS of the service to check")
+	infoLog                = log.New(os.Stderr, "INFO: ", log.Ldate|log.Ltime|log.Lshortfile)
+	failureCount           int
+	runningOS              = runtime.GOOS
+	IPv6FromMetadataServer net.IP
+	eth0                   net.Interface
```

Comment:
can we make this a local and pass it to `hasRouteToLb`, rather than having it global scope?

also can we rename to something like, "directpathNetworkInterface" ? (`eth0` might not <i>always</i> be its real name)